Scripts/DocumentationAutomation/DocumentationAutomation_test.py

```diff
@@ -0,0 +1,84 @@
+import os
+import demistomock as demisto
+
+
+RETURN_ERROR_TARGET = 'DocumentationAutomation.return_error'
+
+
+def test_get_yaml_obj(mocker):
+    from DocumentationAutomation import get_yaml_obj
+    return_error_mock = mocker.patch(RETURN_ERROR_TARGET)
+
+    # sanity
+    mocker.patch.object(demisto, 'getFilePath',
+                        return_value={'path': os.path.join('..', '..', 'Integrations', 'ANYRUN', 'ANYRUN.yml')})
+    data = get_yaml_obj('12345')
+    # error count should not change
+    assert return_error_mock.call_count == 0
+    # call_args last call with a tuple of args list and kwargs
+    assert data['commonfields']['id'] == 'ANYRUN'
+
+    # invalid yml
+    mocker.patch.object(demisto, 'getFilePath',
+                        return_value={'path': os.path.join('..', '..', 'Integrations', 'ANYRUN', 'ANYRUN.py')})
+    get_yaml_obj('234')
+    assert return_error_mock.call_count == 1
```

Comment:
why is it changing now?

Reply:
because i passed a python file and not a yaml